### ISSUE TITLE
Broken gallery example portfolio_analyzer

### ISSUE THREAD

**Issue by cdeil** [holoviz/panel#6408]
If I go to  https://panel-gallery.pyviz.demo.anaconda.com/portfolio_analyzer I currently see an error that the page is not secure plus this error:

```
bokeh-3.3.4.min.js:89 Uncaught Error: Cannot find module '@bokehjs/models/widgets/select'
    at require (bokeh-3.3.4.min.js:89:23)
    at Object._ (panel.min.js:58:55)
    at require (bokeh-3.3.4.min.js:107:21)
    at Object._ (panel.min.js:46:491)
    at require (bokeh-3.3.4.min.js:107:21)
    at Object._ (panel.min.js:45:73)
    at require (bokeh-3.3.4.min.js:107:21)
    at main.register_plugin (bokeh-3.3.4.min.js:147:20)
    at panel.min.js:39:20
````

Is it just me or can you reproduce?

<img width="1624" alt="Screenshot 2024-02-28 at 21 50 07" src="https://github.com/holoviz/panel/assets/852409/4a9711ac-94e7-43f3-9e68-bac89b37ae9d">

**Comment by Hoxbro** [comment]
@philippjfr, I think this will be fixed with a new dev release of 1.4.0.

**Comment by cdeil** [comment]
For me this is working now again.

https://panel-gallery-dev.holoviz.dsp.anaconda.com/portfolio_analyzer

I see one warning in the browser console. Not sure if it's important.

<img width="597" alt="Screenshot 2024-03-16 at 21 10 26" src="https://github.com/holoviz/panel/assets/852409/cb5b44ca-1737-4e19-96fb-e110522a2d6b">
